pkg/util/azureclient/authz/remotepdp/types.go

```diff
@@ -32,6 +32,7 @@ type SubjectAttributes struct {
 	AltSecId         string   `json:"altsecid,omitempty"`
 	IdentityProvider string   `json:"idp,omitempty"`
 	Issuer           string   `json:"iss,omitempty"`
+	ClaimName        string   `json:"_claim_names,omitempty"`
```

Comment:
One last question I guess the s in "_claim_name**s**" is not a typo and is expected, right? Just checking. (most probably it doesn't matter)

Reply:
Right, that's expected.  Here's what's in the CheckAccess document:

![image](https://github.com/user-attachments/assets/1c404f32-875b-48cb-8c7e-f8432c055bac)

Reply:
Perfect. Thanks.